odoo/tests/loader.py

```diff
@@ -74,7 +74,7 @@ def run_suite(suite, module_name=None):
     module.current_test = module_name
     threading.current_thread().testing = True
 
-    results = OdooTestResult()
+    results = OdooTestResult(suite_tests_count=suite.countTestCases())
```

Comment:
Note that this is not accurate when there are subtests and/or tests failing in the `setUpClass`

Reply:
I think the accuracy is not very important here, the developer just wants to have a general idea, (considering that it is just the number of tests and not a time estimate), so IMHO a little inaccuracy does not hurt.

Reply:
Why not making it accurate? :)

Reply:
Because it would be even more complex  😃

Reply:
I don't think the counter should take subtest into account, but what would be the result if a setUpClass fails as you said? My main concern is that the counter should be reach the end: The last test increment should be consistent with the total.

Reply:
> I don't think the counter should take subtest into account

The number of tests failed takes the subtests into account though. You can see this for instance at the end of a test run
> 2 failed, 2 error(s) of 1 tests